src/components/coffeechat.ts

```diff
@@ -0,0 +1,265 @@
+import { Client } from 'discord.js-commando';
+import { Database } from 'sqlite';
+import { openDB } from './db';
+import vars from '../../config/production/vars.json';
```

Comment:
we shouldn't import from production by default, we should use `process.env.NODE_ENV` to determine which config file we pull